src/lib/ngsi/ContextAttribute.h

```diff
@@ -85,6 +85,7 @@ typedef struct ContextAttribute
   std::string  renderAsJsonObject(ConnectionInfo* ciP, RequestType request, const std::string& indent, bool comma, bool omitValue = false);
   std::string  renderAsNameString(ConnectionInfo* ciP, RequestType request, const std::string& indent, bool comma = false);
   std::string  toJson(bool isLastElement, bool types, const std::string& renderMode);
+  std::string  toJsonAsValue(ConnectionInfo* ciP);
```

Comment:
Normally ContextAttribute is rendered as 

    "name": {
       "type": "x",
       "value": "123",
      ...

This new method I added for the case  "GET /v2/entities/EEE/attr/attrrName/value", where only the value is rendered (and as standalone).
I could have added it to **toJson()**, with a new parameter, but it would make that method innecessarily complex.